app/controllers/sessions_controller.rb

```diff
@@ -54,4 +55,17 @@ def fetch_http_referrer
 
     http_referrer.delete_prefix Plek.new.website_root
   end
+
+  def is_valid_level_of_authentication?(value)
+    %w[level0 level1].include? value
+  end
+
+  def is_valid_redirect_path?(value)
```

Comment:
Do we think this will ever end up anywhere else outsdie of `frontend` and `account-api`?
I'm wondering if a Validations module in the personalisations gem might be a better home?

But if it's only ever really going to be these two then that might be premature

Reply:
:+1: For now I can't think where else we'd want to use this validation, but if a third use does crop I agree we should put it in the gem.